xml/System.Data.Common/DbBatch.xml

```diff
@@ -306,11 +337,22 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" />
       </Parameters>
       <Docs>
-        <param name="behavior">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="behavior">An instance of <see cref="T:System.Data.CommandBehavior" />, specifying options for batch execution and data retrieval.</param>
+        <param name="cancellationToken">A token to cancel the asynchronous operation.</param>
+        <summary>Providers should implement this method to provide a non-default implementation for <see cref="Overload:System.Data.Common.DbCommand.ExecuteReader" /> overloads.  
```

Comment:
This is too long for a summary, and most of it should go in the remarks instead.

"The summary succinctly states what a type or member does or represents. All types and members must have summaries, consisting of a short description (optimally one sentence) that conveys the importance of the class or member." - from https://github.com/dotnet/dotnet-api-docs/wiki/Summary.

Reply:
I agree - but this is just a copy-paste from DbCommand (like a lot of this PR). The docs for async APIs in System.Data.Common generally need improvement - I can do a separate pass for all of them, how does that sound?

Reply:
Thanks, sounds good.